docs/who-uses-meson-python.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _who-uses-meson-python:
+
+**************************
+Who uses ``meson-python``?
+**************************
+
+Here's a curated list of projects using ``meson-python``.
+
+
+.. list-table::
+   :widths: 20 80
+
+   * - `SciPy <https://github.com/scipy/scipy>`_
+     - Probably the most complex project using Meson and ``meson-python``.
+       Staring at the build definition too long may cause headaches.
+
+   * - `scikit-image <https://github.com/scikit-image/scikit-image>`_
+     - Another complex project using ``meson-python``.
```

Comment:
I suggest: "A project of medium complexity, with a lot of Cython code"

(no need to repeat "using meson-python" - that's above the table already)